app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav-related.scss

```diff
@@ -1,39 +1,39 @@
 .gem-c-step-nav-related {
   @include govuk-text-colour;
-  border-top: 2px solid $govuk-blue;
-  margin-bottom: $gutter;
+  border-top: 2px solid govuk-colour("blue");
+  margin-bottom: govuk-spacing(6);
 }
 
 .gem-c-step-nav-related__heading {
-  margin-top: $gutter-half;
-  margin-bottom: $gutter-one-third;
-  @include bold-19;
+  margin-top: govuk-spacing(3);
+  margin-bottom: govuk-spacing(2);
+  @include govuk-font(19, $weight: bold);
 }
 
 .gem-c-step-nav-related__links {
-  @include bold-16;
+  @include govuk-font(16, $weight: bold);
   margin: 0;
   padding: 0;
 }
 
 .gem-c-step-nav-related--singular {
-  margin-bottom: $gutter-one-third + 3;
+  margin-bottom: govuk-spacing(2) + 3;
 
   .gem-c-step-nav-related__heading {
-    @include _core-font-generator(19px, 19px, 19px, 1.4, 1.4, false, bold);
-    margin-top: $gutter-two-thirds;
+    @include govuk-font(19, $weight: bold, $line-height: 1.4);
```

Comment:
This replacement might be a bit more complex than this. Currently the heading is 19px at all viewport sizes. With this change it drops to 16px on mobile. I don't remember the reasoning behind this, but I do remember writing it this way very deliberately. Maybe we should discuss offline.

Reply:
We have a few instances of `_core-font-generator` in step-by-step components for which we don't have a replacement mixin. The only workaround that I can see is to overwrite the `govuk-font` on mobile with a `font-size: 19px;`

Reply:
That seems like a good solution.